pom.xml

```diff
@@ -427,8 +427,6 @@
 							<library>resttemplate</library>
 							<generateApiTests>false</generateApiTests>
 							<generateModelTests>false</generateModelTests>
-							<typeMappings><typeMapping>OffsetDateTime=Instant</typeMapping></typeMappings>
```

Comment:
Perchè?

Reply:
Dopo alcuni test, io e Fabrizio Lana abbiamo convenuto che serve che i timestamp vengano generati come OffsetDateTime, così da poter utilizzare il RestTemplate che serializza correttamente le date (altrimenti le date vengono serializzate come long in millisecondi e questo porta in errore la chiamata a SafeStorage).